doc/devel/build-partition-without-eups.rst

```diff
@@ -17,6 +17,7 @@ Build
 *****
```

Comment:
The is a second-level heading, so it needs to use `=` following our style guide, with only an underline.

For more information, see the [RestructuredText Style Guide on headings](https://developer.lsst.io/restructuredtext/style.html#sections).

Reply:
:+1: